### PR TITLE
fixes issue #32

### PR DESCRIPTION
**Pull request by SlavaRa**

### CODE REVIEW

File: src/as3hx/Writer.hx [hunk 1]
```diff
@@ -2565,10 +2565,14 @@ class Writer
                     var rebuiltExpr = EField(EIdent("Std"), "string");
                     rebuiltCall = ECall(rebuiltExpr, [e]);
                 }
+				else if (f == "concat" && params.empty()) {
```
Comment (nadako) [inline src/as3hx/Writer.hx]: there's something wrong with the indetation it seems

Comment (SlavaRa) [inline src/as3hx/Writer.hx]: sorry, fixed

File: src/as3hx/Writer.hx [hunk 2]
```diff
@@ -2565,10 +2565,17 @@ class Writer
                     var rebuiltExpr = EField(EIdent("Std"), "string");
                     rebuiltCall = ECall(rebuiltExpr, [e]);
                 }
+                else if (f == "concat" && params.empty()) {
+                    var type = getExprType(e);
+                    if (type != null && type.indexOf("Array") != -1) {
+                        var rebuildExpr = EField(e, "copy");
+                        rebuiltCall = ECall(rebuildExpr, params);
+                    }
```
Comment (Simn) [inline src/as3hx/Writer.hx]: Don't we need some else-case here? Otherwise we just silently do nothing as far as I can tell.

Comment (SlavaRa) [inline src/as3hx/Writer.hx]: I've done similar to the indexOf. If in the future there will be problems, then I will fix them

**Pull request merged**